test/integration_test/cloud_migration_test.go

```diff
@@ -0,0 +1,80 @@
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func testBasicCloudMigration(t *testing.T) {
```

Comment:
Don't use CloudMigration. The spec and controllers are called Migration. Rename the test file too.